### ISSUE TITLE
Contratação Freelancer

### ISSUE THREAD

**Issue by AmandaMozena** [pontus-vision/pontus-vision#47]
**Resumo:  **
Contratação Freelancer

 **Como um:**
Marketing Pontus

 **Eu quero:  **
Contratar freelancer para frontend do site da Pontus Vision

** Para que:**
Para layouts e necessidades específicas, como divulgação do novo produto SaaS

**Comment by AmandaMozena** [comment]
Estamos trabalhando com webiny e provavelmente conseguiremos seguir sozinhos. Talvez precisaremos de freela somente para layout no website.

**Comment by MrRosario** [comment]
Ola Amanda sou Desenvolvedor Frontend, e estou interessado na vaga

**Comment by AmandaMozena** [comment]
Oi, Eugenio @MrRosario.
Obrigada pelo interesse. Esta tarefa está ainda como projeto interno.
Havendo evolução, entro em contato para batermos um papo.

**Comment by sumitshinde-84** [comment]
Hello,

My name is Sumit Shinde, and I am a passionate and hardworking full-stack developer. I have recently completed my education and am actively seeking job opportunities to kickstart my career. While I don't have any professional experience yet, I have gained valuable skills and knowledge through my education and by contributing to open-source projects.

I am a fast learner and have the ability to pick up new technologies quickly. I am eager to put my skills to use and learn from experienced professionals in the industry. If you know of any relevant job opportunities or have any advice to share, I would greatly appreciate it. Thank you!
